playbooks/roles/ocp-config/templates/install-config.yaml.j2

```diff
@@ -17,7 +17,7 @@ networking:
   clusterNetwork:
   - cidr: 10.128.0.0/14
     hostPrefix: 23
-  networkType: OpenShiftSDN
+  networkType: {{ cni_network_provider }}
```

Comment:
should this be part of role variable with a default (OpenShiftSDN) and documented ?
Also should we use this opportunity to make the hard code values for cidr, prefix etc configurable ?

Reply:
/cc @yussufsh

Reply:
@pravin-dsilva please add the default values in [this file](https://github.com/ocp-power-automation/ocp4-playbooks/blob/master/playbooks/roles/ocp-config/defaults/main/main.yaml).

Variables for cidr, prefix can go in another commit.

Reply:
Added the default value and entry in readme. Please review. Thanks.